common/decklist.cpp

```diff
@@ -500,32 +547,36 @@ bool DeckList::loadFromStream_Plain(QTextStream &in)
             isSideboard = true;
```

Comment:
2 lines above:
The line content is now surely lowercase, so: `if(line.startsWith("sb:")) {`